src/jdk.jpackage/share/classes/jdk/jpackage/internal/AbstractBundler.java

```diff
@@ -53,10 +53,10 @@ public String toString() {
     }
 
     @Override
-    public void cleanup(Map<String, ? super Object> params) {
+    public void cleanup(Map<String, ? super Object> params, boolean includeRuntime) {
```

Comment:
not really true - since in split case there are two calls to cleanup - the first after building the app with includesRuntime false, and the second after building the runtime (as before)